### PR TITLE
[WIP] Main develop branch PR

### PR DESCRIPTION
**Pull request by nakajima-john-shotaro**
## Pull Request Type

- develop

## Description

Merged frontend and backend files.

## Pre-Review Checklist for the Pull Request Author

- [x] Code follows coding-guidelines
- [x] Assign Pull Request to reviewer

## Checklist for the Pull Request Reviewer

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code follows coding-guidelines
- [ ] (Optional) Unit tests have been written for new behavior
- [x] Pull Request title describes the changes

## Post-Review Checklist for the Pull Request Author

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write release notes

### CODE REVIEW

File: aicon/frontend/communication.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from logging import debug
+from flask import render_template, Flask
+from flask_cors import CORS
+
```
Comment (urasakikeisuke) [inline aicon/frontend/communication.py:4]:
```suggestion
from waitress import serve
```

Comment (urasakikeisuke) [reply aicon/frontend/communication.py:4]: https://qiita.com/shimac/items/557dd5e6b3071ba12f05

File: aicon/frontend/static/js/functions.js [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    
+    // 表示するやつ（※後で削除する）
+    // document.getElementById("info").textContent = use_model;
+    // const val = document.getElementById("output")
+    // val.innerText = slider_val
+
+    // 送信するデータ
+    send_data = {
+        model_name : use_model,
+        text : "cute very very long dog. I'm slave on lab",
```
Comment (urasakikeisuke) [inline aicon/frontend/static/js/functions.js:34]: 変更していただけないでしょうか

File: aicon/frontend/static/js/functions.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    var objDef = new $.Deferred();
+    setTimeout(function () {
+        objDef.resolve(msec);
+    }, msec);
+    return objDef.promise();
+}
+
+function communicate(s_data) {
+    $.ajax({
+        url: "http://slack.tasakilab:8081/",
```
Comment (urasakikeisuke) [inline aicon/frontend/static/js/functions.js:54]: ここ柔軟に変更できるようにしてください

File: aicon/frontend/static/js/functions.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+        contentType: "application/json; charset=utf-8",
+    })
+        .done(function (r_data, textStatus, xhr) {
+            console.log("Communication success");
+            // console.log(r_data);
+            console.log(s_data);
+            const tmp_data = JSON.parse(s_data);
+
+            // 通信継続の確認
+            if (r_data["complete"] == false) {
```
Comment (urasakikeisuke) [inline aicon/frontend/static/js/functions.js:69]:
```suggestion
            if (!r_data["complete"] ) {
```

File: aicon/frontend/communication.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+@app.route("/")
+def index():
+    # return "Hello World!"
+    return render_template("aicon.html", title="AIcon", name="AIcon")
+
+
+if __name__ == "__main__":
+    app.run(debug=True, host="0.0.0.0", port=8083, threaded=True)
```
Comment (urasakikeisuke) [inline aicon/frontend/communication.py:17]:
```suggestion
    serve(app, host='0.0.0.0', port=8083)
```

**Review by urasakikeisuke — CHANGES_REQUESTED** [review]
PythonについてはなるべくPEPに従うようにしてください。

**Review by urasakikeisuke — APPROVED** [review]
~LGTM~

File: aicon/frontend/templates/aicon.html [hunk 6]
```diff
@@ -0,0 +1,175 @@
...
+            </form>
+          </div>
+
+          <div class="row">
+            <div class="col s4"></div>
+            <div class="col s6 left-align">
+            <form action="#">
+              <p>
+                <label>
+                  <input id="indeterminate-checkbox" type="checkbox" />
```
Comment (urasakikeisuke) [inline aicon/frontend/templates/aicon.html:85]: ユニークなidに変更してください

File: aicon/frontend/templates/aicon.html [hunk 7]
```diff
@@ -0,0 +1,175 @@
...
+                  <span>Notification</span>
+                </label>
+              </p>
+              <p>
+                <label>
+                  <input id="indeterminate-checkbox" type="checkbox" />
```
Comment (urasakikeisuke) [inline aicon/frontend/templates/aicon.html:91]: ユニークなidに変更してください

**Review by urasakikeisuke — CHANGES_REQUESTED** [review]

File: aicon/frontend/templates/aicon.html (hunk 6 above)
Comment (nakajima-john-shotaro) [reply aicon/frontend/templates/aicon.html:85]: Done. 👍

**Review by urasakikeisuke — APPROVED** [review]

**Review by urasakikeisuke — APPROVED** [review]

**Pull request merged**